src/main/kotlin/org/rust/cargo/RustfmtWatcher.kt

```diff
@@ -46,7 +47,7 @@ class RustfmtWatcher {
         val file = document.virtualFile ?: return false
         if (file.isNotRustFile) return false
         val project = guessProjectForFile(file) ?: return false
-        if (!project.rustfmtSettings.state.runRustfmtOnSave) return false
+        if (!project.rustSettings.rustfmt.runRustfmtOnSave) return false
```

Comment:
Why do I need to call `rustSettings` to get `rustfmt` settings? Why not just `project.rustfmt`?